docs/kleenex/usages.rst

```diff
@@ -181,9 +181,10 @@ some exceptions.
     ``-testbed_file``, "testbed YAML file to load."
     ``-clean_file``, "YAML file(s) containing clean configuration details"
     ``-clean_devices``, "list of devices to clean"
-    ``-clean_image``, "list of clean images per device in format 'device:image_path' or 'device:type:image_path'"
-    ``-clean_platform``, "list of clean images per OS in format 'os:image_path' or 'os:type:image_path'"
-    ``-clean_separator``, "character to separate device/OS from image_path in args 'clean_image' & 'clean_platform'"
+    ``-clean_device_image``, "list of clean images per device in format device:/path/to/image.bin"
```

Comment:
do we still need `kleenex` command? If I'm correct, `kleenex` == `pyats clean` commands.
so simply update all to new syntax with `--` & `-`?

Reply:
Kleenex is deprecated but hasn't been removed yet. You're right that this doc is stale and needs to be updated, and some of the info might need to be moved under the pyats clean docs in the genie repo. However this will involve rewriting this entire file, so maybe it should be a different work item. Maybe it could be done once kleenex command has been actually removed.

Reply:
> maybe this should all be done at once in a future work item

it also works.

Reply:
Created an item for this and put it in the backlog